src/dmd/globals.d

```diff
@@ -153,8 +153,8 @@ struct Param
     bool betterC;           // be a "better C" compiler; no dependency on D runtime
     bool addMain;           // add a default main() function
     bool allInst;           // generate code for all template instantiations
-    bool check10378;        // check for issues transitioning to 10738
-    bool bug10378;          // use pre- https://issues.dlang.org/show_bug.cgi?id=10378 search strategy
+    bool check10378;        // check for issues transitioning to 10738 @@@DEPRECATED@@@ Remove in 2010-05 or later
```

Comment:
2010-05 is a bit past us

Reply:
That's intentional.  It should be removed 1 year from when this is released so it no longer displays a deprecation warning.